serde/src/lib.rs

```diff
@@ -263,28 +233,14 @@ mod lib {
     };
     #[cfg(all(feature = "std", no_target_has_atomic, not(no_std_atomic64)))]
```

Comment:
Why are these imports still here, but not the ones with some of the cfgs disabled?